captum/optim/_core/loss/__init__.py

```diff
@@ -0,0 +1,37 @@
+from captum.optim._core.loss._loss import (  # noqa: F401
```

Comment:
I think that we might to pass `loss_wrapper` here as well for custom modules, like those in the CustomModules tutorial notebook.